osquery/events/pathset.h

```diff
@@ -11,114 +11,130 @@
 #pragma once
 
 #include <mutex>
-#include <set>
 #include <string>
 #include <vector>
 
 #include <boost/noncopyable.hpp>
 #include <boost/tokenizer.hpp>
+#include <boost/utility/string_view.hpp>
 
 #include <osquery/core.h>
 #include <osquery/filesystem.h>
 
 namespace osquery {
 
 /**
- * @brief multiset based implementation for path search.
+ * @brief Vector based implemention for path search.
```

Comment:
Not sure if we need to specific doc. What do you think? ( I mean writing the container type)

Reply:
I leave this to you. What do you suggest?